### PR TITLE
Better ptex glew find

### PR DESCRIPTION
**Pull request by charlesfleche**
### Description of Change(s)

This PR improves the `FindGLEW` and `FindPTex` cmake modules by allowing searching in custom `lib64` directories for GLEW and PTex include subfolder.

### Fixes Issue(s)
-

### CODE REVIEW

**Comment by jilliene** [comment]
Filed as internal issue #USD-5178

File: cmake/modules/FindPTex.cmake [hunk 1]
```diff
@@ -67,6 +67,10 @@ else ()
             "${PTEX_LOCATION}/include/wdas"
             "$ENV{PTEX_LOCATION}/include"
             "$ENV{PTEX_LOCATION}/include/wdas"
+            "${PTEX_LOCATION}/include/Ptex"
+            "${PTEX_LOCATION}/include/wdas"
+            "$ENV{PTEX_LOCATION}/include/Ptex"
+            "$ENV{PTEX_LOCATION}/include/wdas"
```
Comment (c64kernal) [inline cmake/modules/FindPTex.cmake]: It seems like  "${PTEX_LOCATION}/include/wdas" and the ENV version are already included above on lines 67 and 69.

Comment (charlesfleche) [reply cmake/modules/FindPTex.cmake]: Well spotted, thanks. Fixed in another commit.

Comment (c64kernal) [reply cmake/modules/FindPTex.cmake]: That's great, thanks so much, @charlesfleche

**Pull request merged**